src/components/InputField/styles/InputField.ts

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components'
+
+export const InputFieldStyled = styled.div`
+  width: 100%;
+`
+
+export const InputFieldTitle = styled.p.attrs({ colour: 'white' })`
+  margin: 0px;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 50px;
```

Comment:
any reason line height is way bigger than 18px? consider using values like 1 or 1.2?

Reply:
followed styling based on figma file; replaced line-height with relative sizing in 8a9e69fb99032cf3ebc79e5502d5b6a086a45f1e